docs/docs/manual/grelfunctions.md

```diff
@@ -141,6 +141,12 @@ You cannot find or replace nulls with this, as null is not a string. You can ins
 
 Returns the string obtained by replacing a character in s, identified by find, with the corresponding character identified in replace. For example, `"Téxt thát was optícálly recógnízéd".replaceChars("áéíóú", "aeiou")` returns the string “Text that was optically recognized”. You cannot use this to replace a single character with more than one character.
 
+###### replaceEach(s, a find, a replace) {#replaceeach-s-a-find-a-replace}
+
+Returns the string obtained by replacing each element in s of a find array with the corresponding element of a replace array. For example, `"The cow jumps over the moon and moos".replaceEachFromArrays(["oo", "moo"], ["ee", "mee"])` returns the string “The cow jumps over the meen and mees”.
```

Comment:
```suggestion
Returns the string obtained by replacing each element in s of a find array with the corresponding element of a replace array. For example, `"The cow jumps over the moon and moos".replaceEachFromArrays(["th", "moo"], ["ex", "mee"])` returns the string “exe cow jumps over exe meen and mees”.
```

Your other example is ambigious, as once "oo" are converted to "ee", there is no more "woo", because they are already "wee". It defeats the purpuse of multiple replace... What do you think? Make sense @elroykanye?

EDIT: By the way, is it case-sensitive? It could be in the documentation.

Reply:
Oh! That's true 😅
In my mind at the time, this was happening async. Will update as suggested 😁
Thanks

Reply:
> Oh! That's true 😅 In my mind at the time, this was happening async. Will update as suggested 😁 Thanks

He, he, he... But wait! Maybe you should document if it's async or sequencial (and if so, in what axis...) ;-)
This really never ends...

Reply:
It doesn't 😁

Reply:
@elroykanye: sure. But how do you proceed? You search the whole string with the first item, you replace, then you search the replaced string with the second item, and so on? If so, you should say it, because your implemantation could be a single pass algorythm.

Reply:
Yes, it does so sequentially.
```java
// replace each occurrence of search with corresponding element in newReplace
for(int i = 0; i < search.length; i++) {
    str = str.replace(search[i], newReplace[i]);
}
```

Reply:
> EDIT: By the way, is it case-sensitive? It could be in the documentation.

Oh it is not. However, it does not have any support for regex. I think I should update that.

Reply:
> Yes, it does so sequentially.
> (...)

Well, let's keep our user informed for the next years by putting this detail in the Documentation. Fair enough?